critiquebrainz/frontend/templates/review/entity/base.html

```diff
@@ -15,19 +19,33 @@
         <div class="alert alert-warning" role="alert">{{ _('<b>This review has not been published yet!</b> Only you can see it.') }}</div>
       {% else %}
         <em class="text-muted">{{ _('Published on %(date)s', date = review.created|date) }}</em><br />
-        <em class="text-muted">{{ _('Written in %(language)s', language = review.language|language_name) }}</em><br />
-        {% set votes_total = review.votes.positive + review.votes.negative %}
-        <em class="text-muted">
-          {% if votes_total > 0 %}
-            {{ _('%s out of %s people have found this review useful' % (review.votes.positive, votes_total)) }}
-          {% else %}
-            {{ _('This review has not yet been rated') }}
-          {% endif %}
-        </em>
+        {% if review.text %}
+          <em class="text-muted">{{ _('Written in %(language)s', language = review.language|language_name) }}</em><br />
+          {% set votes_total = review.votes.positive + review.votes.negative %}
+          <em class="text-muted">
+            {% if votes_total > 0 %}
+              {{ _('%s out of %s people have found this review useful' % (review.votes.positive, votes_total)) }}
+            {% else %}
+              {{ _('This review has not yet been rated') }}
+            {% endif %}
+          </em>
+        {% endif %}
       {% endif %}
 
-      <p style="word-wrap: break-word; white-space: pre-wrap;">{{ review.text_html|safe }}</p>
-
+      {% if review.rating %}
+        <div style="margin-top: 15px;">
+          <input type="number" class="rating" id="rating" value={{ review.rating }} data-readonly data-inline />
+        </div>
+      {% endif %}
+      {% if review.text %}
+        <!-- Used to display only-text review properly -->
```

Comment:
What is used? Not sure if I understand what issue you are trying to solve there.